lib/src/feature_discovery.dart

```diff
@@ -895,13 +895,20 @@ class _Content extends StatelessWidget {
                     const SizedBox(height: 8.0),
                     description == null
```

Comment:
If you commit this directly, you will need to commit the other ones below as well.
```suggestion
                    if (description == null)
```

Reply:
I don't get it. This won't compile?...

Reply:
This should not have been added like this, the reviews on GitHub are not that great. I wanted to propose to use collection-if here, but the SDK constraints are too low. We could add that later..

Reply:
What do you mean?